rcdt_utilities/nodes/moveit_controller_node.py

```diff
@@ -97,6 +104,34 @@ def move_to_pose(self, pose: PoseStamped) -> bool:
         self.planner.set_goal_state(pose_stamped_msg=pose, pose_link=self.ee_link)
         return self.plan_and_execute()
 
+    def add_object(
+        self, request: AddObject.Request, response: AddObject.Response
+    ) -> None:
+        with self.monitor.read_write() as scene:
+            scene: PlanningScene
+
+            collision_object = request.object
+            collision_object.operation = CollisionObject.ADD
+            collision_object.header.frame_id = "fr3_link0"
+
+            scene.apply_collision_object(collision_object)
+            current_state: RobotState = scene.current_state
+            current_state.update()
+            response.success = True
+            return response
```

Comment:
Deze regels hoeven niet binnen de context manager, beter om ze er buiten te halen

```suggestion
        current_state.update()
        response.success = True
        return response
```